### DISCUSSION TITLE
进行一个助的求

### DISCUSSION THREAD

**Discussion by 99galaxy** [tonquer/picacg-qt#207] [General]
运行了start.exe后提示没有找到几个.py文件,把两个库都安了一遍也無用...
求问怎么解......

**Comment by tonquer** [comment]
什么提示

**Reply by 99galaxy** [reply]
就这个：
Traceback (most recent call last):
  File "start.py", line 12, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "qt_error.py", line 3, in <module>
ImportError: DLL load failed: 找不到指定的程序。

**Reply by tonquer** [reply]
> 就这个： Traceback (most recent call last): File "start.py", line 12, in File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module File "qt_error.py", line 3, in ImportError: DLL load failed: 找不到指定的程序。

你是win7吗？win7的qt版本只能是6.1.3

**Reply by tonquer** [reply]
> 就这个： Traceback (most recent call last): File "start.py", line 12, in File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module File "qt_error.py", line 3, in ImportError: DLL load failed: 找不到指定的程序。

如果是win7 64位，请下载windows7版本运行

**Reply by 99galaxy** [reply]
我说怎么回事……
感谢！
